docs/cassandra.md

```diff
@@ -39,7 +39,7 @@ logging:
       type: cassandra_gc
   service:
     pipelines:
-      apache:
+      cassandra-pipeline:
```

Comment:
Nit: Why the `-pipeline`?

Reply:
We seem to be inconsistent in this, some places its <app>, some places its <app>_pipeline, going to standardize them all to <app>.